### PR TITLE
WIP: add labs script for using pyradiomics with DICOM

### PR DESCRIPTION
**Pull request by fedorov**
Re #434

### CODE REVIEW

**Comment by fedorov** [comment]
@JoostJM I am confused about flak8 errors - local checks pass. Do you know why it behaves differently on CI?

@zhenweishi the updated script is here. Once this PR is merged, it will be ready for your use.

**Comment by fedorov** [comment]
TODO:

- [ ] include pyradiomics version
- [ ] fix flake8 issues

**Comment by fedorov** [comment]
> I am confused about flake8 errors - local checks pass. Do you know why it behaves differently on CI?

I think I understand - I did not take into account the project specific settings in https://github.com/Radiomics/pyradiomics/blob/2b98e8b15a2a8a3574869cb82dc32473591a2e09/.flake8

**Comment by JoostJM** [comment]
> I think I understand - I did not take into account the project specific settings in 

Should not matter if you run flake8 from the root directory.

File: labs/pyradiomics-dcm/pyradiomics-dcm.py [hunk 1]
```diff
@@ -0,0 +1,556 @@
...
+        elif re.match("exponential", prefix):
+            modifiers.append({"modifier": imageTransformationConcept,
+                              "modifierValue": self.makePrivateCode("Exponent transformation")})
+
+        elif re.match("exponential", prefix):
+            modifiers.append({"modifier": imageTransformationConcept,
+                              "modifierValue": self.makePrivateCode("Exponent transformation")})
+
+        # parameterized processing operations
+        elif re.match("wavelet-([HL]{3})", prefix):
```
Comment (fedorov) [inline labs/pyradiomics-dcm/pyradiomics-dcm.py]: Should allow 2 characters in case of forced 2d extraction.

**Comment by zhenweishi** [comment]
Hi, Andrey and Joost

Thanks both. I will continue to work on that.

Best,
Zhenwei

**Comment by JoostJM** [comment]
btw @fedorov, I did a forced-push to remove an unnecessary merge from the history.

**Comment by fedorov** [comment]
I may need to do few adjustments. I found a meaningful place to put pyradiomics processing parameters, but I will need to add things to dcmqi first to allow encoding of those attributes.

**Comment by fedorov** [comment]
@zhenweishi can you tell me when you are planning to do the conversion? There is this one issue about encoding algorithm identification that I would like to fix, and it would help me prioritize things if I know your plans a bit better.

**Pull request merged**